java/client/src/main/java/com/youtube/vitess/client/cursor/Cursor.java

```diff
@@ -40,191 +31,32 @@
  * accessed concurrently without additional synchronization.
  */
 public abstract class Cursor implements AutoCloseable {
-  public abstract long getRowsAffected() throws SQLException;
+  private Map<String, Integer> fieldMap;
 
-  public abstract long getInsertId() throws SQLException;
+  abstract public long getRowsAffected() throws SQLException;
```

Comment:
Done. Was not able to access intellij (which has auto style formatting on save) on my desktop.